sklearn/linear_model/ridge.py

```diff
@@ -1476,17 +1482,38 @@ def fit(self, X, y, sample_weight=None):
         if error:
             best = cv_values.mean(axis=0).argmin()
         else:
-            # The scorer want an object that will make the predictions but
-            # they are already computed efficiently by _RidgeGCV. This
-            # identity_estimator will just return them
-            def identity_estimator():
-                pass
-            identity_estimator.decision_function = lambda y_predict: y_predict
-            identity_estimator.predict = lambda y_predict: y_predict
-
-            # signature of scorer is (estimator, X, y)
-            out = [scorer(identity_estimator, cv_values[:, i], y.ravel())
-                   for i in range(len(self.alphas))]
+
+            if not is_clf:
+
+                # The scorer want an object that will make the predictions but
+                # they are already computed efficiently by _RidgeGCV. This
+                # identity_estimator will just return them
+                def identity_estimator():
+                    pass
```

Comment:
This line is not covered: https://codecov.io/gh/scikit-learn/scikit-learn/compare/06632c0d185128a53c57ccc73b25b6408e90bb89...c381afc9812d75bf6421b8d00539cf642126c824/diff#D2-1492

This is the reason that codecov complain

Reply:
So it could be considered as a false alarm

Reply:
As you had mentioned it as potential false alarm, I am marked it as resolved as of now. Please reopen this, if you can suggest any other option for it.